stac_api/runtime/setup.py

```diff
@@ -14,14 +14,14 @@
     "stac-fastapi.extensions~=5.0",
     "stac-fastapi.pgstac>=5.0.3,<6.0",
     "jinja2>=2.11.2,<4.0.0",
-    "starlette-cramjam>=0.3.2,<0.4",
+    "starlette-cramjam>=0.4.0",  # Compression middleware for non-stac-auth-proxy mode
```

Comment:
```suggestion
```

I think the STAC Auth Proxy defines this as a requirement: https://github.com/developmentseed/stac-auth-proxy/blob/40444cf2cfdd6cb8e660ecd35ce5f03055ca3f7e/pyproject.toml#L18

Reply:
Ah, I see now that the comment states that this is for the non-auth-proxy mode.  However, I don't think we actually use it in the `app.py`

Reply:
https://github.com/NASA-IMPACT/veda-backend/pull/531/files#diff-1efe9954dd21beb72adbdd1429da7ea9db2c9b4c187dfa0bf0c05471c343c428R29 we use it here 😯